### PR TITLE
3D modify and workflow added

### PR DESCRIPTION
**Pull request by tanyh2018**
3D modify and workflow added

### CODE REVIEW

File: geant4_build.sh [hunk 1]
```diff
@@ -10,12 +10,12 @@ mkdir -p $HOME/geant4/build
...
 mkdir -p $HOME/geant4/data 
 # wget https://geant4-data.web.cern.ch/releases/geant4.10.07.p02.tar.gz
-tar xzf geant4.10.07.p02.tar.gz -C $HOME/geant4/src 
+#tar xzf geant4.10.07.p02.tar.gz -C $HOME/geant4/src 
 cd $HOME/geant4/build 
 cmake -DCMAKE_INSTALL_PREFIX=$HOME/geant4/install  -DGEANT4_INSTALL_DATA=ON -DGEANT4_USE_OPENGL_X11=ON -DGEANT4_INSTALL_DATADIR=$HOME/geant4/data -DGEANT4_BUILD_MULTITHREADED=ON -DGEANT4_USE_QT=ON  -DGEANT4_BUILD_TLS_MODEL=global-dynamic  -DGEANT4_USE_PYTHON=ON ../src/geant4.10.07.p02  
 make -j24 
 make install 
-
+cd $HOME/raser-1/
```
Comment (xshi) [inline geant4_build.sh:18]: remove?

File: python/3d/__init__.py [hunk 2]
```diff
@@ -1192,194 +1374,177 @@ def fill_his(model,depth,my_d,my_f,plane):
...
+    elif "2D" in sensor_model:
+        x_v=2500
+        y_v=2500
+    else:
+        print("sensor model is wrrong")
+        sys.exit()        
+    if "scan" in sensor_model:
+        print("change_para:%s"%change_para)
+        output_path = output + "/"+change_para
+        os.system("mkdir %s -p"%(output_path))
```
Comment (xshi) [inline python/3d/__init__.py:1493]: better to use os.mkdir(path)

File: geant4_build.sh (hunk 1 above)
Comment (tanyh2018) [reply geant4_build.sh:18]: The code is used to install geant4-pybind, and I will note these codes

Comment (tanyh2018) [reply geant4_build.sh]: @xshi  these codes are used to install geant4-pybind, I have noted these codes.

File: python/3d/__init__.py (hunk 2 above)
Comment (tanyh2018) [reply python/3d/__init__.py]: Have changed

**Pull request merged**